users/gotham/process_records.c

```diff
@@ -0,0 +1,66 @@
+#include "gotham.h"
+
+uint16_t clipboard_timer;
+
+__attribute__((weak)) bool process_record_keymap(uint16_t keycode, keyrecord_t *record) { return true; }
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+#ifdef KEYLOGGER_ENABLE
+    xprintf("KL: kc: %u, col: %u, row: %u, pressed: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed);
+#endif  // KEYLOGGER_ENABLE
+    if (process_record_keymap(keycode, record)
+#ifdef OLED_DRIVER_ENABLE
+        && process_record_user_oled(keycode, record)
+#endif
+#ifdef ENCODER_ENABLE
+        && process_record_user_encoder(keycode, record)
+#endif
+#ifdef THUMBSTICK_ENABLE
+        && process_record_user_thumbstick(keycode, record)
+#endif
+    ) {
+        switch (keycode) {
+            case KC_QWERTY ... KC_NORMAN:
+                if (record->event.pressed) {
+                    set_single_persistent_default_layer(keycode - KC_QWERTY);
+                }
+                return false;
+                break;
+
+            case KC_MAKE:
+                if (!record->event.pressed) {
+                    SEND_STRING("make " QMK_KEYBOARD ":" QMK_KEYMAP
+#if (defined(BOOTLOADER_DFU) || defined(BOOTLOADER_LUFA_DFU) || defined(BOOTLOADER_QMK_DFU))
+                                ":dfu BOOTLOADER=atmel-dfu"
+#elif defined(BOOTLOADER_HALFKAY)
+                                ":teensy BOOTLOADER=halfkay"
+#elif defined(BOOTLOADER_CATERINA)
+                                ":avrdude BOOTLOADER=caterina"
+#endif
```

Comment:
you can replace this with `:flash`.  And then add the bootloader option based on the bootloader part. 

either way works, though.

Reply:
Ah, I missed that. Fixed.